test/sushi/sushi-weth_DELETE_ME.js

```diff
@@ -0,0 +1,234 @@
+const { send } = require("@openzeppelin/test-helpers");
+const BigNumber = require("bignumber.js");
+const IERC20 = artifacts.require("@openzeppelin/contracts/token/ERC20/IERC20.sol:IERC20");
+
+//const Strategy = artifacts.require("");
+const Strategy = artifacts.require("SushiStrategyULMainnet_SUSHI_WETH_DELETE_ME");
+const IFeeRewardForwarder = artifacts.require("IFeeRewardForwarderV6");
+
+//This test was developed at blockNumber 13885848
+
+// Vanilla Mocha test. Increased compatibility with tools that integrate Mocha.
+describe("Sushi: SUSHI:WETH DELETE ME", function () {
+	let accounts;
+
+	// external contracts
+	let underlying;
+
+	// external setup
+	let underlyingWhale = "0x88B521167CC35A22B51EA5caDD7DbCd4cc2Cbc57";
+	let weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
+	let feeForwarderAddr = "0x153C544f72329c1ba521DDf5086cf2fA98C86676";
+
+	let sushi = "0x6B3595068778DD592e39A122f4f5a5cF09C90fE2";
+
+	let sushiDexBytes32 = "0xcb2d20206d906069351c89a2cb7cdbd96c71998717cd5a82e724d955b654f67a";
+	let bancorDexBytes32 = "0x4bf11b89310db45ea1467e48e832606a6ec7b8735c470fff7cf328e182a7c37e";
+
+	// parties in the protocol
+	let governance;
+	let farmer1;
+
+	// numbers used in tests
+	let farmerBalance;
+
+	// Core protocol contracts
+	let controller;
+	let vault;
+	let strategy;
+
+	async function setupExternalContracts() {
+		underlying = await IERC20.at("0x795065dCc9f64b5614C407a6EFDC400DA6221FB0");
+		console.log("Fetching Underlying at: ", underlying.address);
+	}
+
+	async function setupBalance() {
+		let etherGiver = accounts[9];
+		// Give whale some ether to make sure the following actions are good
+		await web3.eth.sendTransaction({ from: etherGiver, to: underlyingWhale, value: 10e18 });
+		await web3.eth.sendTransaction({ from: etherGiver, to: governance, value: 10e18 });
+
+		farmerBalance = await underlying.balanceOf(underlyingWhale);
+		await underlying.transfer(farmer1, farmerBalance, { from: underlyingWhale });
+	}
+
+	before(async function () {
+		governance = "0xf00dD244228F51547f0563e60bCa65a30FBF5f7f";
+		accounts = await web3.eth.getAccounts();
+
+		farmer1 = accounts[1];
+
+		// impersonate accounts
+		await impersonates([governance, underlyingWhale]);
+
+		await setupExternalContracts();
+		// whale send underlying to farmers
+		await setupBalance();
+
+		await setupExternalContracts();
+		[controller, vault, strategy] = await setupCoreProtocol({
+			"existingVaultAddress": null,
+			"strategyArtifact": Strategy,
+			"strategyArtifactIsUpgradable": true,
+			"underlying": underlying,
+			"governance": governance,
+			//{ "sushi": [sushi, weth] } does already exist, but added for completness
+			"liquidation": [{ "sushi": [sushi, weth] }],
+		});
+
+		let feeForwarder = await IFeeRewardForwarder.at(feeForwarderAddr);
+		let path = [sushi, weth, addresses.FARM];
+		let dexes = [sushiDexBytes32, bancorDexBytes32];
+		await feeForwarder.configureLiquidation(path, dexes, { from: governance });
```

Comment:
I think this would already be set, or is there an error when running without this?

Reply:
I removed it and it works without the feeForwarder setting in the test.